src/view/reviewManager.ts

```diff
@@ -297,7 +297,7 @@ export class ReviewManager implements vscode.DecorationProvider {
 
 			const comment = await this._prManager.createCommentReply(this._prManager.activePullRequest, text, thread.threadId);
 			thread.comments.push({
-				commentId: comment.id,
+				commentId: String(comment.id),
```

Comment:
since we're using `toString` elsewhere and this should never be null or undefined, would prefer to use `toString` for consistency